code-challenges.test.js

```diff
@@ -44,14 +73,80 @@ const votes2 = { upVotes: 2, downVotes: 33 }
 
 // Pseudo code:
 
+// 1.) Create a function that takes in a parameter(value) 
+
+const voteCounter = (value) => {
+
+// 2.) Have the return value as an agruments and use dot notations to access the values of the upvote and downvote keys and subtract the upvote from the downvote. 
+
+  return value.upVotes - value.downVotes
+}
+
+
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
+describe("newArray", () => {
+  it("returns one array with no duplicate values.", () => {
+    // const dataTypesArray1 = ["array", "object", "number", "string", "Boolean"]
+    // const dataTypesArray2 = ["string", "null", "Boolean", "string", "undefined"]
+    expect(dataTypesArray1, dataTypesArray2).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
```

Comment:
You should have your function call here

Reply:
Are you referring to lines 100 and 101?

Reply:
Line 94 in your expect statement. Right now you just have the 2 variables without a function call.